pages/linux/lxi.md

```diff
@@ -0,0 +1,24 @@
+# lxi
+
+> Tool for controling LXI compatible instruments such as oscilloscopes.
+> More information: <https://github.com/lxi-tools/lxi-tools>.
+
+- Discover LXI devices on available networks:
+
+`lxi discover`
+
+- Capture a screenshot using plugin autodetection:
+
+`lxi screenshot --address {{10.42.1.20}}`
+
+- Capture a screenshot using specifying a plugin:
+
+`lxi screenshot --address {{10.42.1.20}} --plugin {{rigol-1000z}}`
+
+- Send SCPI command to an instrument:
+
+`lxi scpi --address {{10.42.1.20}} {{"*IDN?"}}`
```

Comment:
Maybe make the ip more generic, 192.168.*.100 or 10.0.0.100

Reply:
Usually we use a place holder like `ip_address`, also the `"` should be outside the tokens: `"{{*IDN?}}"`

Reply:
https://github.com/tldr-pages/tldr/pull/6174/commits/188ab49aa22917135c067941e7f3a5d4ce0fe9ed